authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.4 on 2019-01-08 14:41
+# Generated by Django 2.1.4 on 2019-01-18 07:38
```

Comment:
We shouldn't update the migration files

Reply:
Okay noted

Reply:
Can you just remove the change made to this file?